static/js/src/advantage/users/components/TableView/TableView.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { render, screen, waitFor } from "@testing-library/react";
+
+import { User } from "../../types";
+import { mockUser } from "../../mockData";
+import TableView from "./TableView";
+
+it("displays user details in a correct format", () => {
+  const testUser: User = {
+    id: "1",
+    email: "user@ecorp.com",
+    role: "Admin",
+    createdAt: "2020-01-10T10:00:00Z",
+    lastLoginAt: "2021-02-15T13:45:00Z",
+  };
+
+  render(<TableView users={[testUser]} />);
+
+  screen.getByText("user@ecorp.com");
+  screen.getByText("Admin");
+  screen.getByText("10/01/2020");
+  screen.getByText("15/02/2021");
+});
+
+it("allows to edit only a single user at a time", async () => {
+  const users: User[] = [
+    { ...mockUser, id: "1", email: "karen@ecorp.com", role: "Billing" },
```

Comment:
Do you mean by clone to "copy and paste the entire object"? Because I'm already using the spread operator which clones each value of the object (as they're all primitives - strings) instead of pointing to reference.

I think I might use a JSON file for mock data instead. 🤔